src/libs/actions/MemoryOnlyKeys/exposeGlobalMemoryOnlyKeysMethods/index.js

```diff
@@ -0,0 +1,12 @@
+import * as MemoryOnlyKeys from '../MemoryOnlyKeys';
+
+const exposeGlobalMemoryOnlyKeysMethods = () => {
+    window.enableMemoryOnlyKeys = () => {
+        MemoryOnlyKeys.enable();
+    };
+    window.disableMemoryOnlyKeys = () => {
+        MemoryOnlyKeys.disable();
+    };
```

Comment:
What is it for? Hand-debugging via console, or what?

Reply:
We're just [adding typing](https://github.com/Expensify/App/pull/32164#discussion_r1424192764) for this part and I'm wondering if it's actually beneficial to extend the `Window` interface, or maybe we don't need any typing for this if we don't intend to use it programmatically.

Reply:
Yes, it's a JS console thing. And no, it shouldn't really be used in the code anywhere. So I agree there is no point in getting crazy with the types for this.